src/main/java/gregtech/common/metatileentities/multi/electric/generator/MetaTileEntityLargeTurbine.java

```diff
@@ -100,7 +107,21 @@ protected void addDisplayText(List<ITextComponent> textList) {
             FluidStack fuelStack = ((LargeTurbineWorkableHandler) recipeMapWorkable).getInputFluidStack();
             int fuelAmount = fuelStack == null ? 0 : fuelStack.amount;
 
+            // energy output (recipeMapWorkable.getRecipeEUt())
+            // recipe fluid (recipeMapWorkable.getPreviousRecipe().fluidInputs)
+            // recipe eut (recipeMapWorkable.getPreviousRecipe().EUt)
+            // recipe duration (recipeMapWorkable.getPreviousRecipe().duration)
+            // recipeMapWorkable.getPreviousRecipe().EUt * recipeMapWorkable.getPreviousRecipe().duration / recipeMapWorkable.getPreviousRecipe().fluidInputs.get(0).amount
```

Comment:
What is the state of this commented out code?

Reply:
this is just me figuring out what functions to call when using the debugger, I'll remove it